puffin/src/global_profiler.rs

```diff
@@ -127,7 +127,7 @@ impl GlobalProfiler {
     }
 
     /// Reports some profiling data. Called from [`ThreadProfiler`].
-    pub(crate) fn report(
+    pub fn report(
```

Comment:
This needs to be made public so that `GlobalProfiler::report()` can be called by our custom `ThreadProfiler.reporter` that we assign.